README.md

```diff
@@ -175,6 +176,77 @@ overwrite the corresponding index in the existing array, however an index that h
 set to any value (i.e. empty values in a sparse array) will not overwrite the corresponding
 index in the existing array, even though value at both indexes evaluates to undefined.
 
+### Preventing Default Recursive Merge
+Occasionally you may want to avoid persisting values from earlier in the application state. 
+This is especially true for single page applications where you may not want outdated information 
+in the data model when routing between pages.
+
+To prevent the default recursive merging behavior, a flag can be passed in adjacent 
+to the object(s) or array(s) you wish to prevent merging. To do so, add a truthy '_clear' attribute 
+to the message with the key/value pair(s) you are targeting. Here are some examples:
+
+<table>
```

Comment:
Could we switch this to the same direction as other existing tables in this doc?  i.e. column names at the top, 1 row = 1 example (rather than the reverse).

Reply:
I went ahead and switched it to match the other tables!

I noticed that for a single row it was pretty unreadable due to how complicated the example model was so I added in shorter examples from the tests to demonstrate the very basic behavior and left the more complicated one in as an example of using the _clear flag deeper in the model.